cmd/peer-s3-client.go

```diff
@@ -171,7 +171,9 @@ func (sys *S3PeerSys) GetBucketInfo(ctx context.Context, bucket string, opts Buc
 	}
 
 	for i, err := range errs {
-		if err == nil {
+		// bucketInfos:[local,remote...]
+		// errs:[nil,remote...]
+		if err == nil && i != 0 {
```

Comment:
This is not needed @jiuker if local succeeded we always return local on purpose but make sure we verify quorum.

We do not need it to be a different remote endpoints output.

Reply:
Then the loop is meaningless, because surely the first one must be nil @harshavardhana

Reply:
No we need to deduce quorum, if quorum fails we do not return the local disk.